zerver/actions/realm_settings.py

```diff
@@ -486,3 +487,62 @@ def do_send_realm_reactivation_email(realm: Realm, *, acting_user: Optional[User
         language=language,
         context=context,
     )
+
+
+def do_send_realm_deactivation_email(realm: Realm, acting_user: Optional[UserProfile]) -> None:
+    # if realm is not deactivated, but changed the domain
+    if not acting_user:
+        return
+
+    event_time = timezone_now()
+    RealmAuditLog.objects.create(
+        realm=realm,
+        acting_user=acting_user,
+        event_type=RealmAuditLog.REALM_DEACTIVATION_EMAIL_SENT,
+        event_time=event_time,
+        extra_data=orjson.dumps(
+            {
+                RealmAuditLog.ROLE_COUNT: realm_user_count_by_role(realm),
+            }
+        ).decode(),
+    )
+    send_to_deactivating_owner = True
+    context = {
+        "realm_uri": realm.uri,
+        "realm_name": realm.name,
+        "event_date": event_time.date(),
+        "acting_user_name": acting_user.full_name,
+        "send_to_deactivating_owner": send_to_deactivating_owner,
+    }
+    language = realm.default_language
+    owners = realm.get_human_owner_users()
+
+    deactivating_owner = acting_user
+    deactivating_owner_id = deactivating_owner.uuid
+    other_owners = owners.exclude(uuid=deactivating_owner_id)
+    # owners_set = set(owners)
```

Comment:
Please remove these unused commented codes.

Reply:
oh, that was for the reviewer to check using either exclude function or set.